### PR TITLE
:new: :robot: cohorts: Add OTTAA Project (@OTTAA-Project)

### PR DESCRIPTION
**Pull request by jwflory**
This commit adds a new team profile to the Data Science & AI cohort. This
builds on the work done for unicef/inventory-hugo-theme#13 by
@Idadelveloper as part of unicef/inventory#78.

Screenshot of summary card for new team profile.

Screenshot of full team profile.

### CODE REVIEW

**Comment by jwflory** [comment]
Hi @hectoritr, @gonojuarez, & @OTTAAProject team, I am tagging you here for an initial review of the OTTAA Project team profile to be published on the UNICEF Open Source Inventory site! This is a new feature on the O.S. Inventory to highlight your open source repositories and points of engagement. The goal is to make your efforts more visible and discoverable.

I need your help to verify the following information before this is officially published:

1. Verify that product summary/elevator pitch, social media links, and other company info are correct.
2. Confirm which [UN Sustainable Development Goals](https://sdgs.un.org/goals) align with your open-source product.

You can leave feedback as a comment on this Pull Request or reach out to me over email. If you are GitHub-savvy, you can also leave a proper review to "sign off" on this Pull Request too, by either approving it or requesting changes. To find the review feature, you can follow these screenshots below to start a review:

<details>

**Screenshot of the "Files changed" menu, where the review feature can be found.**

Screenshot of the "Files changed" menu, where the review feature can be found.
</details>

<details>

**Click the _Review changes_ button to add your review.**

Showing the "Review changes" menu in the upper-right corner, where feedback can be left once the menu is opened.
</details>

<details>

**Screenshot of the review interface.**

Screenshot of the review interface.
</details>

Let me know if you have questions!

**Comment by jwflory** [comment]
Hey @hectoritr, a friendly ping to take a look at this pull request and confirm the information or suggest any changes when you get a chance. Have a good Friday. 🙂

**Comment by hectoritr** [comment]
Hi Justin! Yeap my bad, I've already read it and wrote some comments but didn't press the Comment button.

File: content/cohorts/ds-ai/ottaa-project.en.adoc [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+categories: "team"
+tags: ["team"]
+type: profiles
+
+description: An Alternative Augmentative Communication (AAC) system, intended for people with speech disabilities.
+country: Chile
+date: 2020-09-25
+github: https://github.com/OTTAA-Project
+linkedin: https://www.linkedin.com/company/ottaa-project/
+twitter: https://twitter.com/OTTAAProject
```
Comment (hectoritr) [inline content/cohorts/ds-ai/ottaa-project.en.adoc:13]: We are very strong on Facebook and Instagram, would be nice to have those handlers as well. 
Both are @OTTAAProject

File: content/cohorts/ds-ai/ottaa-project.en.adoc [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+website: https://www.ottaaproject.com/
+community: ""
+logo: /inventory/cohorts/logo-ottaa-project.png
+sdgs:
+    - 4
```
Comment (hectoritr) [inline content/cohorts/ds-ai/ottaa-project.en.adoc:18]: -SDG 3 , 4 , 10

File: content/cohorts/ds-ai/ottaa-project.en.adoc [hunk 3]
```diff
@@ -0,0 +1,34 @@
...
+    - 6
+    - 7
+    - 8
+    - 9
+
+---
+
+OTTAA Project is the first real intelligent AAC platform that uses environmental data and a powerful AI algorithm that in combination with a pictogram based communication code allows users to create sentences and communicate effectively every single day.
+This AI-powered alternative and augmentative communication tool allow users with disabilities to communicate, express their feelings and basic needs.
+The team directly responds to the challenge of fostering social inclusion for people with disabilities and ensuring effective communication for those with complex communication needs.
```
Comment (hectoritr) [inline content/cohorts/ds-ai/ottaa-project.en.adoc:34]: The team directly responds to the challenge of fostering social inclusion for people with disabilities and ensuring effective communication for those with complex communication needs. By assuring better communication in an academic or medical setting the overall outcome of those interactions is much stronger and better catered to specific needs.

**Review by hectoritr — CHANGES_REQUESTED** [review]
Here are my comments! :)

**Comment by jwflory** [comment]
@hectoritr Thanks for the review! Let me see if I can work Facebook and Instagram into the templates. I'll follow up on the rest here soon. Have a nice weekend!

**Review by Idadelveloper — COMMENTED** [review]
@jwflory are we to add  Facebook and Instagram? Also, you're yet to include SDGs 3 & 10 in this PR.

**Comment by jwflory** [comment]
@Idadelveloper Thanks, I'll adjust the SDGs in another commit.

For Facebook and Instagram, I was thinking to add support for these sites into the theme. It doesn't look like it will be hard but I did not have a chance to dig into it yet.

File: content/cohorts/ds-ai/ottaa-project.en.adoc (hunk 2 above)
Comment (jwflory) [reply content/cohorts/ds-ai/ottaa-project.en.adoc:19]: Resolved in commit 3b26cb44151b7969e43c2a1fafa916faeede880a.

**Comment by jwflory** [comment]
This has been pending for some time. The only thing this Pull Request is blocked on is social media for Facebook and Instagram. In the interest of getting the useful information we already have here published, I will merge now and we can follow up later once more social media channels are added. I also added this as a user story in a [project requirements list](https://docs.google.com/spreadsheets/d/1vcISpk3AEWTgEj3AZfogV5VLJnRUSv6fZsAt85_u_Ro/edit?usp=sharing) for our upcoming intern.

**Pull request merged**